components/dashboard/forms/TradeEntryForm.tsx

```
@@ -0,0 +1,112 @@
+
+      if (!res.ok) {
+        throw new Error(`Failed to submit trade: ${res.statusText}`);
+      }
+
+      const saved = await res.json();
+      console.log("✅ Trade saved:", saved);
+
+      setSuccessMessage("Trade saved successfully!");
+      setErrorMessage("");
+      form.reset(); // Clear form
+    } catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)